test-framework-docker/executor/src/main/java/com/containersolutions/mesoshelloworld/executor/FrameworkExecutor.java

```diff
@@ -0,0 +1,81 @@
+package com.containersolutions.mesoshelloworld.executor;
+
+import org.apache.mesos.Executor;
+import org.apache.mesos.ExecutorDriver;
+import org.apache.mesos.MesosExecutorDriver;
+import org.apache.mesos.Protos.ExecutorInfo;
+import org.apache.mesos.Protos.FrameworkInfo;
+import org.apache.mesos.Protos.SlaveInfo;
+import org.apache.mesos.Protos.Status;
+import org.apache.mesos.Protos.TaskID;
+import org.apache.mesos.Protos.TaskInfo;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Adapted from https://github.com/apache/mesos/blob/0.22.1/src/examples/java/TestExecutor.java
+ */
+public class FrameworkExecutor implements Executor {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(FrameworkExecutor.class);
+
+    public static final String RESPONSE_STRING = "Hello world";
+
+    private Thread thread;
+
+    public static void main(String[] args) throws Exception {
+        MesosExecutorDriver driver = new MesosExecutorDriver(new FrameworkExecutor());
+        if (driver.run() != Status.DRIVER_STOPPED) {
+            throw new RuntimeException("Mesos Executor Driver is not stopped");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS00112)